utils/config.go

```diff
@@ -448,6 +452,8 @@ func GenerateClientConfig(c *model.Config, diagnosticId string, license *model.L
 	props["DiagnosticsEnabled"] = strconv.FormatBool(*c.LogSettings.EnableDiagnostics)
 
 	props["PluginsEnabled"] = strconv.FormatBool(*c.PluginSettings.Enable)
+	props["EnableTimezoneSelection"] = strconv.FormatBool(*c.DisplaySettings.EnableTimezoneSelection)
+	props["SupportedTimezones"] = strings.Join(c.SupportedTimezones, ",")
```

Comment:
This is going to make the client config a lot larger and could potentially slow down first load as a result. The client only needs to know this when the user is selecting a timezone, right? If so I wonder if it's better to isolate this into it's own REST API endpoint that gets called when the user opens their account settings to the timezone page

Reply:
It could potentially cause a slow down on first load. 

let me get some metrics to see what the difference makes with this change

Reply:
@jwilander I'll work on getting this refactored into an endpoint.  May take some time as I'll have to update the clients but won't be a huge change.

Also, be on a lookout for the redux library supporting this. I'll add it there as well.

thanks for your review 😄

Reply:
Sounds good :)